server.js

```diff
@@ -18,6 +20,30 @@ const app = next({ dev });
 
 const handle = app.getRequestHandler();
 
+const checkStatus = response => response.ok ? response.json() : Promise.reject(response.json());
+
+function sendContactToHubSpot({ firstname, lastname, email, phone }) {
```

Comment:
@dmytropanontko , could it be an arrow-function?

Reply:
@epaminond yes